docs/pressable.md

```diff
@@ -0,0 +1,175 @@
+---
+id: pressable
+title: Pressable
+---
+
+Pressable is a Core Component wrapper that can detect various types of interaction states on any of it's defined children.
+
+```jsx
+<Pressable onPress={onPressFunction}>
+  <Text>I'm pressable!</Text>
+</Pressable>
+```
+
+## Example
+
+```
+import * as React from 'react';

```

Comment:
It would be nice to avoid `*` in import, can you change this line to `import React, { useState } from 'react';`?